extensions/markdown-language-features/src/preview/security.ts

```diff
@@ -60,7 +60,8 @@ export class ExtensionContentSecurityPolicyArbiter implements ContentSecurityPol
 	}
 
 	public shouldDisableSecurityWarnings(): boolean {
-		return this._workspaceState.get<boolean>(this._should_disable_security_warning_key, false);
+		// MEMBRANE: disable markdown security warning
+		return this._workspaceState.get<boolean>(this._should_disable_security_warning_key, true);
```

Comment:
I just realized the warning only shows up on localhost, not prod. So I'll revert this.